### PR TITLE
Update navBar.astro

### PR DESCRIPTION
**Pull request by vionya**
i made the code better

### CODE REVIEW

**Comment by Sakooooo** [comment]
thanks fwsedirju ghnvrdeito:3

File: src/components/navBar.astro [hunk 1]
```diff
@@ -27,19 +27,7 @@
...
- a:active {
-     color: white;
-     text-decoration: none !important;
- }
- a:visited {
-     color: white;
-     text-decoration: none !important;
- }
- a:link {
+ a:focus, a:active, a:visited, a:link {
```
Comment (MichailiK) [inline src/components/navBar.astro:30]: This could be made even DRY'er:
`a:is(:focus, :active, :visited, :link) { ... }`

Comment (Sakooooo) [reply src/components/navBar.astro:30]: I just got off my computer oh my god im just gonna merge and pray I guess

Comment (Sakooooo) [reply src/components/navBar.astro:30]: oh my god I just turned off my computer ill just leave it as is for now until something changes

Comment (vionya) [reply src/components/navBar.astro:30]: michaili i hate you

**Pull request merged**